src/Screen.js

```diff
@@ -163,6 +163,16 @@ class Navigator {
   }
 }
 
+class Button {
```

Comment:
The more I think about this, the more I wonder if this can be accomplished using `Touchable`s, instead of creating a view that gets its own `onPress` defined.